lib/slack/web/api/too_many_requests_error.rb

```diff
@@ -0,0 +1,21 @@
+module Slack
+  module Web
+    module Api
+      class TooManyRequestsError < ::Faraday::Error
```

Comment:
It may be confusing that this error is namespaced under `Slack::Web::Api::Errors` while the existing `Slack::Web::Api::Error` class is not namespaced (and is very close to `Slack::Web::Api::Errors`).  Do you still want the change?